File: code-recipes/php/test/reports/RequestAndRetrieveReportRecipeTest.php
<?php

namespace Test\reports;

use Src\reports\RequestAndRetrieveReportRecipe;
use Test\util\RecipeTestCase;

class RequestAndRetrieveReportRecipeTest extends RecipeTestCase
{
    protected function setUp(): void
    {
        $this->recipe = new RequestAndRetrieveReportRecipe();
        $this->responses = [
            "reports-createReport",
            "reports-getReportDocument",
        ];
    }
}

File: code-recipes/java/src/test/java/reports/RequestAndRetrieveReportRecipeTest.java
package reports;

import util.RecipeTest;

import java.util.List;

public class RequestAndRetrieveReportRecipeTest extends RecipeTest {

    protected RequestAndRetrieveReportRecipeTest() {
        super(
                new RequestAndRetrieveReportRecipe(),
                List.of(
                        "reports-createReport",
                        "reports-getReportDocument"
                )
        );
    }
}
